src/NexyCrypt.php

```diff
@@ -299,7 +320,7 @@ private function signedPostRequest($uri, array $payload)
      *
      * @return ResponseInterface
      */
-    private function request($method, $uri, array $options = [])
+    private function request($method, $uri, array $options = ['verify' => false])
```

Comment:
Maybe we can just include the required SSL certificate files and we don't need the additional package?